src/styles/blocks/card.scss

```diff
@@ -0,0 +1,97 @@
+.card {
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px ;
+  transition: all 300ms;
+
+  &-picture {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto;
+  }
+
+  &-content {
+    width: 166px;
+    margin-top: 40px;
+
+    &__info {
+      padding-bottom: 16px;
+
+      &-name {
+        font-weight: 500;
+        color: $main-text-color;
+        padding-bottom: 4px;
+      }
+
+      &-code {
+        font-size: 10px;
+        font-weight: 400;
+        line-height: 14px;
+        color: $secondary-text-color;
+      }
+    }
+
+    &__stars {
+      @include centeringBlocks;
+      padding-bottom: 24px;
+    }
+
+    &__reviews {
+      font-size: 10px;
+      font-weight: 400;
```

Comment:
fix it everywhere

Reply:
.page {
  font-family: Roboto, Helvetica, sans-serif;
  font-size: 12px;
  line-height: 18px;
}
it's default value
I am wrong?

Reply:
font-weight: 400 - is default only that part